stdlib/public/core/ContiguousArrayBuffer.swift

```diff
@@ -274,6 +274,7 @@ public struct _ContiguousArrayBuffer<Element> : _ArrayBufferProtocol {
 
   /// Call `body(p)`, where `p` is an `UnsafeMutableBufferPointer`
   /// over the underlying contiguous storage.
+  @discardableResult
   public mutating func withUnsafeMutableBufferPointer<R>(
```

Comment:
Please drop this one, for the same reason as you explained on `withUnsafeMutableBufferPointer`.